packages/server/src/fhir/operations/patientsummary.test.ts

```diff
@@ -0,0 +1,270 @@
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .send({ resourceType: 'Organization' });
+    expect(orgRes.status).toBe(201);
+    const organization = orgRes.body as Organization;
+
+    // Create practitioner
+    const practRes = await request(app)
+      .post(`/fhir/R4/Practitioner`)
+      .set('Authorization', 'Bearer ' + accessToken)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .send({ resourceType: 'Practitioner' });
+    expect(practRes.status).toBe(201);
+    const practitioner = practRes.body as Practitioner;
+
+    // Create patient
+    const res1 = await request(app)
+      .post(`/fhir/R4/Patient`)
+      .set('Authorization', 'Bearer ' + accessToken)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .send({
+        resourceType: 'Patient',
+        name: [{ given: ['Alice'], family: 'Smith' }],
+        address: [{ use: 'home', line: ['123 Main St'], city: 'Anywhere', state: 'CA', postalCode: '90210' }],
+        telecom: [
+          { system: 'phone', value: '555-555-5555' },
+          { system: 'email', value: 'alice@example.com' },
+        ],
+        managingOrganization: createReference(organization),
+      } satisfies Patient);
+    expect(res1.status).toBe(201);
+    const patient = res1.body as Patient;
+
+    // Create observation
+    const res2 = await request(app)
+      .post(`/fhir/R4/Observation`)
+      .set('Authorization', 'Bearer ' + accessToken)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .send({
+        resourceType: 'Observation',
+        status: 'final',
+        code: { coding: [{ system: LOINC, code: '12345-6' }] },
+        subject: createReference(patient),
+        performer: [createReference(practitioner), createReference(organization)],
+      } satisfies Observation);
+    expect(res2.status).toBe(201);
+
+    // Create condition
+    // This condition references the patient twice, once as subject and once as asserter
+    // This is to test that the condition is only returned once
+    const res3 = await request(app)
+      .post(`/fhir/R4/Condition`)
+      .set('Authorization', 'Bearer ' + accessToken)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .send({
+        resourceType: 'Condition',
+        code: { coding: [{ system: LOINC, code: '12345-6' }] },
+        asserter: createReference(patient),
+        subject: createReference(patient),
+        recorder: createReference(practitioner),
+      } satisfies Condition);
+    expect(res3.status).toBe(201);
+
+    // Execute the operation
+    const res4 = await request(app)
+      .get(`/fhir/R4/Patient/${patient.id}/$summary`)
+      .set('Authorization', 'Bearer ' + accessToken);
+    expect(res4.status).toBe(200);
+    const result = res4.body as Bundle;
+    expect(result.type).toBe('document');
+    expect(result.entry?.[0]?.resource?.resourceType).toBe('Composition');
+    expect(result.entry?.[1]?.resource?.resourceType).toBe('Patient');
```

Comment:
Can we check that the resources show up in the right sections of the composition?